pkg/apis/kf/v1alpha1/app_lifecycle.go

```diff
@@ -180,7 +181,7 @@ func (status *AppStatus) PropagateServiceAccountStatus(serviceAccount *corev1.Se
 func (status *AppStatus) PropagateRouteStatus(bindings []QualifiedRouteBinding, routes []Route, undeclaredBindings []QualifiedRouteBinding) {
 	var rs []AppRouteStatus
 	var conditions []apis.Condition
-	var urls []string
+	urls := make(sets.String)
```

Comment:
I think we should be using this one: "k8s.io/apimachinery/pkg/util/sets" which is already elsewhere in the codebase

Reply:
Done, thanks